Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsEvent.swift

```diff
@@ -66,6 +66,7 @@ extension Analytics.Event {
         }
         
         enum MessageType: String, Codable {
+            case analytics                          = "ANALYTICS"
```

Comment:
what is this used for?

Reply:
Nowhere at the moment, I removed it. It was used while debugging to send an event with how many events were synced, and how many were still left locally.